src/AMAMM.sol

```diff
@@ -76,6 +76,8 @@ contract AMAMM is IAmAmm {
 
         address msgSender = LibMulticaller.senderOrSigner();
 
+        _userBalance[msgSender] += _getDeposit(id, _epoch);
```

Comment:
This looks like minting money from thin air lol. I think it should call the deposit function here which would increment `u`serBalances and transfer bitToken.